src/components/WelcomeScreen.tsx

```diff
@@ -0,0 +1,90 @@
+import { InputField } from '@components/InputField';
+import * as React from 'react'
+
+export interface Props {
+  hasAccount: boolean
+  isIncorrectPassword: boolean
+  walletName: string
+}
+
+let logo
+if (process.env.IS_WEB) {
+  logo = require('../logo2.svg')
+}
+
+export const WelcomeScreen: React.SFC<Props> = (props) => (
+  <div className="container u-padding-left-sm-2 u-padding-right-sm-2" style={{ maxWidth: '540px' }}>
+    <header className="u-margin-top-xl vertical-spaced level">
+      <figure>
+        <img src={logo} alt="Kinesis logo" />
+      </figure>
+      <div className="heading u-margin-top-xl">
+        <h2 className="is-size-3 has-text-weight-semibold has-text-grey-lighter">Welcome to the</h2>
+        <h1 className="is-size-1 has-text-weight-bold has-text-primary">future of money</h1>
+      </div>
+    </header>
+    {props.hasAccount ? <HasAccount {...props} /> : <NoAccount />}
+  </div >
+)
+
+export class HasAccount extends React.Component<Props> {
+  getInitials(walletName: string): string {
+    const [firstName, surName] = walletName.split(' ')
+    const firstInitial = firstName.charAt(0)
+    const secondInitial = surName ? surName.charAt(0) : ''
+    return `${firstInitial}${secondInitial}`.toUpperCase()
+  }
+
+  render() {
+    return (
+      <div>
+        <section className="level-left u-margin-top-xl">
```

Comment:
I feel like you might be battling against Bulma a bit here, as this `level-left` should probably be nested within a `level`?